### ISSUE TITLE
[UX] User "locked" in the http://www.userfriendly.tech/core/authorize.php page

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#2580]
![backdrop-update_manager-authorize_page-no_back_or_cancel](https://cloud.githubusercontent.com/assets/2423362/23982052/018a7098-0a5f-11e7-82fe-964d155dd503.png)

I think that we should have this be inside a modal. Alternatively, we should add a "cancel" or "back" button. ...but the modal would give us a "close" button for free.

**Comment by jlfranklin** [comment]
👍 for a back, cancel, or similar when there is an insecure action warning on the page!

Not sold on the modal part, but then the Overlay module was the first I turned off on every D7 site.  Modal interfaces in web sites just mess with my natural tendency to hit the back button in the browser.

**Comment by klonos** [comment]
Yes, but the D7 overlay was abusing the purpose of modals. It was placing the entire admin backend on a modal. The right reaction on modals is to hit the "close" button or the "cancel" link. I understand that we are creatures of habit, but would you hit the browser "back" button on say the block config dialogs?

**Comment by klonos** [comment]
...it would be nice if we supported the ESC keyboard mouse for closing dialogs.
